polyfill/lib/ecmascript.mjs

```diff
@@ -358,13 +359,20 @@ export function RejectTemporalLikeObject(item) {
   }
 }
 
+export function CanonicalizeTimeZoneOffsetString(offsetString) {
+  const offsetNs = ParseTimeZoneOffsetString(offsetString);
+  return FormatTimeZoneOffsetString(offsetNs);
+}
+
 export function ParseTemporalTimeZone(stringIdent) {
```

Comment:
This function doesn't correspond with any operation in https://tc39.es/proposal-temporal/ AFAICT; perhaps it should not be exported?